client/my-sites/customer-home/index.js

```diff
@@ -9,9 +9,19 @@ import page from 'page';
 import { navigation, siteSelection, sites } from 'my-sites/controller';
 import home, { maybeRedirect } from './controller';
 import { makeLayout, render as clientRender } from 'controller';
+import { trackNavigationStart } from 'lib/performance-tracking';
 
 export default function () {
-	page( '/home', siteSelection, sites, makeLayout, clientRender );
+	page( '/home', trackNavigationStart( 'home' ), siteSelection, sites, makeLayout, clientRender );
 
-	page( '/home/:siteId', siteSelection, maybeRedirect, navigation, home, makeLayout, clientRender );
+	page(
+		'/home/:siteId',
+		trackNavigationStart( 'home' ),
```

Comment:
When this middleware is run it will call `LUX.init()`. This marks the `start` point of the loading process.